x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -235,6 +235,11 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
         max_doc_count: column.params.orderBy.maxDocCount,
       }).toAst();
     }
+
+    const shardSize = column.params.accuracyMode
+      ? Math.max(1000, column.params.size * 1.5 + 10)
```

Comment:
@flash1293 the why behind this math is a little unclear. Could you suggest a code comment to add next to it?

Reply:
@andrewctate Correct, this is about aligning with Elasticsearch. I don't think it's required to put a mechanism in place to keep it in sync - this will make sure the Lens behavior stays the same even if Elasticsearch changes behavior in this regard so it gives us extra flexibility.